tools/pubsys/src/kit/publish_kit/mod.rs

```diff
@@ -0,0 +1,213 @@
+use crate::Args;
+use buildsys_config::DockerArchitecture;
+use clap::Parser;
+use log::{debug, info, trace};
+use pubsys_config::InfraConfig;
+use regex::Regex;
+use snafu::{ensure, OptionExt, ResultExt};
+use std::fs::read_dir;
+use std::path::{Path, PathBuf};
+use tokio::process::Command;
+
+/// Takes a local kit built using buildsys and publishes it
+/// to a vendor specified in Infra.toml
+#[derive(Debug, Parser)]
+pub(crate) struct PublishKitArgs {
+    /// Location of the local kit
+    #[arg(long)]
+    kit_path: PathBuf,
```

Comment:
nit: For argument parsing-related structs, I believe we tend to newline separate arguments.